app/css/animator.css

```diff
@@ -53,6 +53,47 @@ a {
   text-align: center;
 }
 
+/* ========== CONFIRM BOXES ========== */
+
+#confirm-container {
+    position: absolute;
+    height: 100vh;
+    width: 100vw;
+    background-color: rgba(23, 23, 23, 0.5);
+    z-index: 4;
+    text-align: center;
+}
+
+#confirm-container.hidden { display: none; }
+
+#confirm {
+    position: relative;
+    top: calc(50vh - 10em);
+    display: inline-block;
+    width: 20em;
+    background: #000;
+    padding: 0 1em 1em 1em;
+    border: solid 1px #D9D9D9;
```

Comment:
Per spec ([MDN reference](https://developer.mozilla.org/en-US/docs/Web/CSS/border)), `border` shorthand order is `width style color` (e.g., this should read `1px solid #D9D9D9`).